treeherder/model/models.py

```diff
@@ -1251,6 +1251,23 @@ def elastic_search_insert(self):
             return es_line
 
 
+class Group(models.Model):
+    """
+    The test harness Group.  Sometimes this is a Manifest, but not always.
+    Not to be confused with JobGroup which is Treeherder specific.
+    """
+    id = models.BigAutoField(primary_key=True)
+    name = models.CharField(max_length=200, unique=True)
```

Comment:
I understand that newer versions of windows (8 and newer) allow 1024, but older versions allow just 260.  I admit, I picked 200 just thinking it was "long enough".  I'll change it to 260 for now.  I'd kind of be surprised if we went longer than that, but maybe we will.  1024 seems like overkill though.  Do you have an opinion on the best size?